test/rqg/mzcompose.py

```diff
@@ -0,0 +1,181 @@
+        "--duration",
+        type=int,
+        help="Run the Workload for the specifie amount of time in seconds",
+    )
+    parser.add_argument(
+        "--debug", action="store_true", help="Run the RQG With RQG_DEBUG=1"
+    )
+    parser.add_argument(
+        "workloads", nargs="*", default=None, help="run specified Workloads"
+    )
+    args = parser.parse_args()
+
+    c.up("rqg", persistent=True)
+
+    if args.workloads is not None and len(args.workloads) > 0:
+        workloads_to_run = [w for w in WORKLOADS if w.name in args.workloads]
+    else:
+        workloads_to_run = [w for w in WORKLOADS if not w.disabled]
+
+    assert (
+        len(workloads_to_run) > 0
+    ), f"No matching workloads found (args was {args.workloads})"
+
+    for workload in workloads_to_run:
+        print(f"--- Running workload {workload.name}: {workload.__doc__} ...")
+        run_workload(c, args, workload)
+
+
+def run_workload(c: Composition, args: argparse.Namespace, workload: Workload) -> None:
+    participants: list[Service] = [
+        Materialized(
+            name="mz_this",
+            ports=["16875:6875", "16877:6877"],
+            image=f"materialize/materialized:{args.this_tag}"
+            if args.this_tag
+            else None,
+            use_default_volumes=False,
+        ),
+    ]
+
+    psql_urls = ["postgresql://materialize@mz_this:6875/materialize"]
+
+    if workload.reference_implementation == ReferenceImplementation.MZ:
+        participants.append(
+            Materialized(
+                name="mz_other",
+                image=f"materialize/materialized:{args.other_tag}"
+                if args.other_tag
+                else None,
+                ports=["26875:6875", "26877:6877"],
+                use_default_volumes=False,
+            )
+        )
+        psql_urls.append("postgresql://materialize@mz_other:6885/materialize")
+
+    elif workload.reference_implementation == ReferenceImplementation.POSTGRES:
+        participants.append(Postgres(ports=["15432:5432"]))
+        psql_urls.append("postgresql://postgres:postgres@postgres/postgres")
+    else:
+        assert False
+
+    files = (
+        ["dbt3-ddl.sql", "dbt3-s0.0001.dump"]
+        if workload.dataset == Dataset.DBT3
+        else ["simple.sql"]
+    )
+
+    dsn2 = (
+        [f"--dsn2=dbi:Pg:{workload.reference_implementation.dsn()}"]
+        if workload.reference_implementation is not None
+        else []
+    )
+
+    duration = args.duration if args.duration else workload.duration
+    assert duration is not None
+
+    env_extra = {"RQG_DEBUG": "1"} if args.debug else {}
+
+    with c.override(*participants):
+        try:
+            c.up(*[p.name for p in participants])
+
+            for file in files:
+                for psql_url in psql_urls:
+                    print(f"--- Populating {psql_url} with {file} ...")
+                    c.exec("rqg", "bash", "-c", f"cat conf/mz/{file} | psql {psql_url}")
```

Comment:
```suggestion
                    c.exec("rqg", "bash", "-c", f"psql {psql_url} -f cat conf/mz/{file}")
```
Although this could also be done via `c.sql` I think?

Reply:
Not right now, as individual lines in some of the SQL files will fail with "X already exists" or "no permission to drop schema default" and other such. That would be fatal in for `c.sql()`  but `psql` tolerates it. I will try to clean up the `.sql` files in a follow-up commit.